File: framework-global-id/src/test/java/com/github/caijh/framework/globalid/MistGlobalIdServiceTest.java
package com.github.caijh.framework.globalid;

import com.github.caijh.framework.global.id.service.GlobalIdService;
import com.github.caijh.framework.global.id.service.MistGlobalIdServiceImpl;
import org.junit.jupiter.api.AfterEach;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

class MistGlobalIdServiceTest {

    @BeforeEach
    public void setUp() {

    }

    @AfterEach
    public void tearDown() {

    }


    @Test
    void generate() {
        GlobalIdService globalIdService = new MistGlobalIdServiceImpl();
        for (long i = 0; i < 1000; i++) {
            long id = globalIdService.generate(i);
            System.out.printf(String.valueOf(id));
        }
    }

}

File: framework-global-id/src/test/java/com/github/caijh/framework/globalid/MistGlobalIdServiceTest.kt
package com.github.caijh.framework.globalid

import com.github.caijh.framework.globalid.service.MistGlobalIdServiceImpl
import org.junit.jupiter.api.AfterEach
import org.junit.jupiter.api.BeforeEach
import org.junit.jupiter.api.Test

internal class MistGlobalIdServiceTest {

    @BeforeEach
    fun setUp() {
    }

    @AfterEach
    fun tearDown() {
    }

    @Test
    fun generate() {
        val globalIdService = MistGlobalIdServiceImpl()
        for (i in 1L..1000L) {
            val id = globalIdService.generate(i)
            println(id)
        }
    }

}
